PSMDB-990 Release notes 4.0.28-23

nastena1606 opened the pull request:
Updated copyright year

modified:   source/conf.py
	modified:   source/copyright.rst
	modified:   source/release_notes/index.rst
new file: source/release_notes/4.0.28-13.rst

nastena1606 committed:
PSMDB-990 Release notes 4.0.28-23

Updated copyright year

modified:   source/conf.py
	modified:   source/copyright.rst
	modified:   source/release_notes/index.rst
new file: source/release_notes/4.0.28-13.rst

sandraromanchenko commented on source/release_notes/4.0.28-13.rst:
```diff
@@ -0,0 +1,20 @@
+
+* `SERVER-62065 <https://jira.mongodb.org/browse/SERVER-62065>`_ - Added the ``repairShardedCollectionChunksHistory`` command to restore ``history`` fields for some chunks. This aims to fix broken snapshot reads and distributed transactions.
+* `SERVER-54064 <https://jira.mongodb.org/browse/SERVER-54064>`_ - Added periodic clean up of logical sessions cache on arbiters
+* `SERVER-45953 <https://jira.mongodb.org/browse/SERVER-45953>`_ - Exempted internal replication oplog readers from acquiring read tickets in order to avoid deadlocks
```
PSMDB-950 should be also included

sandraromanchenko commented on source/release_notes/4.0.28-13.rst:
```diff
@@ -0,0 +1,20 @@
+:Date: February 15, 2022
+:Installation: `Installing Percona Server for MongoDB <https://www.percona.com/doc/percona-server-for-mongodb/4.0/install/index.html>`_
+
+Percona Server for MongoDB 4.0.28 is an enhanced, open source, and highly-scalable database that is a
```
I'm confused, we did switch from "open source" to "source-available", didn't we?)

igorsol commented on source/release_notes/4.0.28-13.rst:
```diff
@@ -0,0 +1,20 @@
+Percona Server for MongoDB 4.0.28 is an enhanced, open source, and highly-scalable database that is a
+fully-compatible, drop-in replacement for MongoDB 4.0.28 Community Edition.
+
+It is rebased on MongoDB 4.0.28 Community Edition and supports MongoDB 4.0.28 protocols and drivers.
```
We have always used "It is based ..." wording. Now it is "rebased". Is this an intended change?

nastena1606 replied:
Yep. Peer suggested this wording for cases when we introduce no changes/bug fixes of our own. Yet since we do, I removed it

igorsol replied:
Ok

nastena1606 committed:
Updated install tarball steps

kwilktx commented:
Please add the following "Reminder" to the top section of the release notes and then also list 2 additional impactful bugs in the "Release Highlights" section.

**Reminder**  MongoDB version 4.0 is scheduled to go End of Life April 30, 2022.
Please plan your upgrades accordingly. Below are 2 links that provide more details around Percona’s software and services life cycle policies:
https://www.percona.com/services/policies/percona-software-support-lifecycle
https://www.percona.com/services/policies/percona-services-lifecycle-policy

**Release Highlights**
https://jira.mongodb.org/browse/SERVER-59929
Fixed issue from versions 4.0.26 and below that caused slower update and insert times related to splitChunk and moveChunk operations.

https://jira.mongodb.org/browse/SERVER-55648
Resolved issues related to failures in retryable writes in case of shutdowns

kwilktx commented:
Also please add the following clarifications to the "Upgrading Percona Server for MongoDB" section found here:
https://62150b5fcc4e5cb00fa1f378--psmdb-docs-40.netlify.app/install/upgrade-from-mongodb.html

**Change this:**

An in-place upgrade is done with existing data in the server. Generally speaking, this is stopping the mongod service, removing the old packages, installing the new server and starting it with the same db data directory. An in-place upgrade is suitable for most environments, except the ones that use ephemeral storage and/or host addresses.

**To This:**
An in-place upgrade is done by keeping the existing data in the server. It involves changing out the MongoDB binaries. Generally speaking, the steps include stopping the mongod service, removing the old binaries, installing the new server versions binaries and restarting the mongod service with the same db data directory. An in-place upgrade is suitable for most environments except the ones that use ephemeral storage and/or host addresses.

nastena1606 committed:
Updated in-place upgrade description

modified:   source/install/upgrade-from-mongodb.rst
	modified:   source/release_notes/4.0.28-23.rst

nastena1606 commented:
Updated following Kimberly's comments
